egress/http-proxy/egress-http-proxy.sh

```diff
@@ -90,4 +91,4 @@ sed -e 's/^/  /' ${CONF}
 echo ""
 echo ""
 
-exec squid -N
+exec squid -N -f /tmp/squid.conf
```

Comment:
We should reference `"${CONF}"`.

Reply:
/lgtm cancel

Let's reference the shell variable -- https://github.com/openshift/images/pull/162#pullrequestreview-1977153552.

Reply:
Done, and I've changed `${CONF}` to `"${CONF}"` elsewhere in the script.